readme.md

```diff
@@ -21,15 +21,15 @@ npm install hafas-rest-api-client
 ```js
 import createClient from 'hafas-rest-api-client'
 
-const vbbClient = createClient('https://v5.vbb.transport.rest', {
+const vbbClient = createClient('https://v6.vbb.transport.rest', {
 	// Please pass in a User-Agent header to let the providers of the API endpoint understand how you're using their API.
 	userAgent: 'my awesome project',
 })
 
-const res = await vbbClient.journeys('900000003201', '900000024101', {results: 1})
+const res = await vbbClient.journeys('900100705', '900096197', {results: 1})
 ```
 
-`hafas-rest-api-client` is a client for [`hafas-rest-api@3`](https://www.npmjs.com/package/hafas-rest-api/v/3.8.0) APIs. Check their individual API docs for all supported parameters.
+`hafas-rest-api-client` is a client for [`hafas-rest-api@6`](https://www.npmjs.com/package/hafas-rest-api/v/6.0.0-alpha.1) APIs. Check their individual API docs for all supported parameters.
```

Comment:
It's very confusing, but `v6.{bvg,db,vbb}.transport.rest` actually use `hafas-rest-api@5` underneath:

https://github.com/derhuerst/vbb-rest/blob/e84abb37ccec3d01b60033974d730e3f5429484e/package.json#L34

```suggestion
`hafas-rest-api-client` is a client for [`hafas-rest-api@5`](https://www.npmjs.com/package/hafas-rest-api/v/5.1.3) APIs. Check their individual API docs for all supported parameters.
```